test/syscalls/linux/proc_net.cc

```diff
@@ -54,6 +62,193 @@ TEST(ProcSysNetIpv4Sack, CanReadAndWrite) {
   EXPECT_EQ(buf, to_write);
 }
 
+PosixErrorOr<uint64_t> get_metric(std::string snmp, const char *type, const char *item) {
+  int idx = -1;
+
+  for (auto const& line : absl::StrSplit(snmp, '\n')) {
+    if (!absl::StartsWith(line, type))
+	    continue;
+
+    std::vector<std::string> fields =
+        absl::StrSplit(line, ' ', absl::SkipWhitespace());
+    EXPECT_TRUE(!fields.empty());
+
+    if (idx == -1) {
+      for (size_t i = 1; i < fields.size(); i++) {
+        if (fields[i].compare(item) == 0) {
```

Comment:
clang-tidy gave this suggestion

do not use 'compare' to test equality of strings; use the string equality operator instead
http://clang.llvm.org/extra/clang-tidy/checks/readability-string-compare.html

Reply:
Done. Thanks!